### DISCUSSION TITLE
[Idea] USB cradle and rechargable.

### DISCUSSION THREAD

**Discussion by bademux** [joeycastillo/Sensor-Watch#223] [Ideas]
Hi,
It will be nice to have cradle for SW, we have 4 external contacts for plastic cases:
3 metal buttons (please see images) + backplate. Backplate can be mounted upside down to expose GND.
Basically we have USB without a case modification.

It will be great to implement charging.
Have wired connection to board without case modification.

I see some problems:
if board is connected to USB and battery present, then battery will h
ave current on its contacts. Maybe supercapasitor can be used?
I'm not sure about static electricity.

note:
photos represent kind of PoC: cooper adhesive tape is too soft for daily use.

<a href="https://user-images.githubusercontent.com/1610049/222928389-eb940954-093d-4f4a-ade7-3770bee7f3fa.jpg"><img src="https://user-images.githubusercontent.com/1610049/222928389-eb940954-093d-4f4a-ade7-3770bee7f3fa.jpg" width="300"></a>
<a href="https://user-images.githubusercontent.com/1610049/222928349-5673666c-6082-4874-babc-ca6f6bc98d72.jpg"><img src="https://user-images.githubusercontent.com/1610049/222928349-5673666c-6082-4874-babc-ca6f6bc98d72.jpg" width="300"></a>

**Comment by trash-anger** [comment]
### Great idea! 💡
The other approach could be to make a custom CNC back with a NFC chip that could be used both to load the device and to transfert data from and to the device.

**Comment by nkempe** [comment]
Hello from the future!

This is a really awesome idea!
Are the buttons electrically isolated (insulated?) from the watch case?
Are there watch cases where this would or would not work?

Surprisingly I don't have a Casio watch just yet, but I'll look into this when I get one.

Assuming that the USB is only really practical to use if you can press the reset button, we might need a way to do or mimic that too.
Next to the battery power thing of course.

Still, I like this idea, really creative. :)

**Reply by bademux** [reply]
> Are the buttons electrically isolated (insulated?) from the watch case?
Yes. Casio F-91W case is made of plastic. traces made of cooper adhesive tape

**Reply by nkempe** [reply]
If there is a part of metal from the button that does not move much, perhaps you could use thin copper wire like used in coils in relays. Flat cables could be another more permanent option.

The wire is probably not the issue, the voltage might be.
